### ISSUE TITLE
Donut graph can't handle widely different values for 2 series

### ISSUE THREAD

**Issue by rstudner** [chartist-js/chartist#612]
<img width="253" alt="screen shot 2016-02-16 at 9 39 41 pm" src="https://cloud.githubusercontent.com/assets/209970/13098218/01133a22-d4f6-11e5-9359-1b5ec472f603.png">
<img width="271" alt="screen shot 2016-02-16 at 9 38 08 pm" src="https://cloud.githubusercontent.com/assets/209970/13098222/0787776a-d4f6-11e5-81cd-777b35c65331.png">
<img width="272" alt="screen shot 2016-02-16 at 9 40 05 pm" src="https://cloud.githubusercontent.com/assets/209970/13098224/09924ec2-d4f6-11e5-8284-a0c77a60b3e9.png">

As you can see, the chart 'breaks down' as the 2nd series # increases.

```
      series: [{
        value: lowValue,
      }, {
        value: highValue,
      }],
```

as lowValue stays at 4, and highValue goes from 7180 to 7200 to 7220.. the donut stops being able to handle drawing itself :)

And to note, it is only when the 2nd series ('highValue') grows too large relative to the first.
If this is reversed:

```
      series: [{
        value: 7220,
      }, {
        value: 4,
      }],
```

It renders the donut fine.

**Comment by hansmaad** [comment]
Seems to be same as here: https://github.com/gionkunz/chartist-js/issues/527

**Comment by gionkunz** [comment]
This was fixed in 0.9.6

**Comment by gionkunz** [comment]
Can you check with the latest version?

**Comment by rstudner** [comment]
Will test it out in about 1 hour and report back :)

**Comment by rstudner** [comment]
100% of my bar chart graphs broke / must have been API changes.  So I can't use 0.96 just yet.

**Comment by rstudner** [comment]
0.96 causes other problems:  https://github.com/gionkunz/chartist-js/issues/621

**Comment by gionkunz** [comment]
#621 is fixed and 0.9.7 is published
